### ISSUE TITLE
[TOUCH] UX Suggestion: Create skins that make the keyboards appear more like native keyboards

### ISSUE THREAD

**Issue by matsbla** [keymanapp/keyman#1114]
I think that it would make it easier to make people use a new keyboard if it is similar to something they are already used to. Maybe we could make it become possible to create keyboard skins? And then by default give them a look & feel similar to the native keyboard on the mobile they already use.

**Comment by mcdurdin** [comment]
Yes, we'd like to do that -- both make keyboard skins that keyboard developers can distribute, and allow end users to customise the skins themselves. This is under consideration for Keyman 11.
